src/sentry/workflow_engine/processors/action.py

```diff
@@ -85,3 +93,37 @@ def filter_recently_fired_workflow_actions(
     filtered_actions = actions.filter(id__in=actions_to_include | actions_without_statuses_ids)
 
     return filtered_actions
+
+
+def get_available_action_integrations_for_org(organization: Organization) -> list[RpcIntegration]:
+    providers = [
+        handler.provider_slug
+        for handler in action_handler_registry.registrations.values()
+        if hasattr(handler, "provider_slug")
+    ]
+    return integration_service.get_integrations(
+        status=ObjectStatus.ACTIVE,
+        org_integration_status=ObjectStatus.ACTIVE,
+        organization_id=organization.id,
+        providers=providers,
+    )
+
+
+def get_notification_plugins_for_org(organization: Organization) -> list[PluginService]:
```

Comment:
Why do we have this function twice? It's in `src/sentry/rules/actions/notify_event_service.py` as well but it looks like this one is actually used

Reply:
oop... that is a mistake 🫣